tests/e2e/tidbcluster/tidbcluster.go

```diff
@@ -1498,6 +1498,58 @@ var _ = ginkgo.Describe("[tidb-operator] TiDBCluster", func() {
 		oa.UpgradeTidbClusterOrDie(&clusterConfig)
 		oa.CheckTidbClusterStatusOrDie(&clusterConfig)
 	})
+
+	ginkgo.It("TiDB PD TiKV separate pv", func() {
+
```

Comment:
No need to add e2e as e2e test load is much high now.

Reply:
ok.

Reply:
What about just update an existing case to cover this case and no extra test duration would be introduced.